src/utils/request-utils.ts

```diff
@@ -22,9 +22,9 @@ export const convertToURL = (url: string): HTMLAnchorElement | null => {
     return location
 }
 
-export const isUrlMatchingRegex = function (url: string, pattern: string): boolean {
+export const isMatchingRegex = function (value: string, pattern: string): boolean {
```

Comment:
nit: feels more like string utils than request utils

especially now it's more obviously got nothing to do with URLs

Reply:
makes sense to move it now